client/my-sites/checkout/checkout/secure-payment-form.jsx

```diff
@@ -125,11 +124,7 @@ export class SecurePaymentForm extends Component {
 				if ( this.getInitialCard() ) {
 					newPayment = storedCardPayment( this.getInitialCard() );
 				} else if ( ! get( this.props.transaction, 'payment.newCardDetails', null ) ) {
-					if ( this.shouldUseStripeElements ) {
```

Comment:
If we remove this, as you suggested in the description, we should remove the definition as well (on line 104 above).